tests/end_to_end/client.py

```diff
@@ -50,14 +50,7 @@ def runCommand(self, command):
             "serverUrl": self.serverUrl,
             "protocolVersion": protocol.version, })
         splits = shlex.split(clientCmdLine)
-        try:
-            subprocess.check_call(
-                splits,
-                stdout=self.outFile,
-                stderr=self.errFile)
-        except subprocess.CalledProcessError as error:
-            self.printDebugInfo(command)
-            raise error
+        subprocess.check_call(splits, stdout=self.outFile, stderr=self.errFile)
```

Comment:
Please revert this change, and add a comment/follow up issue if there is a good reason to remove the try block.